secondLargest.js

```diff
@@ -0,0 +1,22 @@
+/**
+*   Return the second largest number in the array.
+*   @param {Number[]} nums - An array of numbers.
+*   @return {Number} The second largest number in the array.
+**/
+function getSecondLargest(nums) {
```

Comment:
If I give the input array as : 
-21 , -2158963 , 0 ,21578963257896, -29

what will be the out put from your code ??

Reply:
getting 0 as output

Reply:
your logic gives wrong output for input :   [-21,-29,-57,-60]

Modify your logic